internal/app/auth_server.go

```diff
@@ -199,6 +208,13 @@ func login(ctx context.Context, cfg *ServerConfig, storage op.Storage, userUseca
 	}
 }
 
+func logout(_ context.Context, _ *ServerConfig, _ op.Storage, _ interfaces.User) func(ctx echo.Context) error {
+	return func(ec echo.Context) error {
```

Comment:
This method is nothing to do?

Reply:
I made like this just in case we decided to implement a sessions for the auth server.

Reply:
Great